howdoi/howdoi.py

```diff
@@ -148,12 +165,23 @@ def _extract_links(html, search_engine):
 def _get_search_url(search_engine):
     return SEARCH_URLS.get(search_engine, SEARCH_URLS['google'])
 
+def _detect_block(page):
+    for indicator in BLOCK_INDICATORS:
+        if page.find(indicator) != -1:
+            return True
+
+    return False
 
 def _get_links(query):
     search_engine = os.getenv('HOWDOI_SEARCH_ENGINE', 'google')
     search_url = _get_search_url(search_engine)
 
     result = _get_result(search_url.format(URL, url_quote(query)))
+    if _detect_block(result):
+        print('[ERROR] Encountered a temporary block due to automated request '
```

Comment:
How about:

Unable to find an answer because the search engine blocked the request. Please wait a few minutes or select a different search engine.

Reply:
Disagree. It's an error and should be indicated as such.

Reply:
If you want to include the error statement, then let's go with `[ERROR] Unable to find an answer because the search engine temporarily blocked the request Please wait a few minutes or select a different search engine.`

Reply:
ab18e0df0219c68f689731667e8a26966029414c